Examples/package-info/Package.swift

```diff
@@ -1,22 +1,25 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.5
 
 import PackageDescription
 
 let package = Package(
     name: "package-info",
     platforms: [
-        .macOS(.v10_15),
+        .macOS("10.15.4"),
```

Comment:
Since this was an interesting minimum deployment to target (due to some Foundation API improvements), should we add this as an enum?  I know we don't usually do that for patch releases but I don't think that's policy, and this one in particular has semantic use.

Reply:
this was just to align it to the version used in SwiftPM, not  sure the genesis

Reply:
maybe we should go to latest instead?

Reply:
Ah, I see.  I don't think it matters which one we use here for the example — latest would be fine, I think.

It just occurred to me that others wanting the Swiftier Foundation FileManager APIs would want 10.15.4 or newer, and that might justify a new enum case.  Maybe we don't yet have enough of them — with an index that would be easy to scan for.

Reply:
no strong option, but going as high as possible is good to make use of new APIs, best practices etc